src/thermo/PlasmaPhase.cpp

```diff
@@ -8,9 +8,15 @@
 #include "cantera/thermo/Species.h"
 #include "cantera/base/global.h"
 #include "cantera/numerics/funcs.h"
+#include "cantera/kinetics/Kinetics.h"
+#include "cantera/kinetics/Reaction.h"
+#include "cantera/kinetics/KineticsFactory.h"
+#include "cantera/kinetics/ElectronCollisionPlasmaRate.h"
 
 namespace Cantera {
 
+static const double gamma = sqrt(2 * ElectronCharge / ElectronMass);
```

Comment:
Rather than `static const double`, this would be better as just `const double` in an unnamed namespace.